addons/survey/static/src/js/survey_result.js

```diff
@@ -255,7 +255,7 @@ publicWidget.registry.SurveyResultChart = publicWidget.Widget.extend({
     },
 
     _getDoughnutChartConfig: function () {
-        var quizz_score = this.$el.data("quizz_score") || 0.0;
+        var scoring_percentage = this.$el.data("scoring_percentage") || 0.0;
```

Comment:
Should be camelCased ideally but whatever.

Reply:
Grep and replace, so ... for another iteration of result cleaning.